UPGRADE-4.4.md

```diff
@@ -123,6 +123,50 @@ TwigBridge
 
  * Deprecated to pass `$rootDir` and `$fileLinkFormatter` as 5th and 6th argument respectively to the
    `DebugCommand::__construct()` method, swap the variables position.
+   
+TwigBundle
+----------
+
+ * Deprecated default value `twig.controller.exception::showAction` of the `twig.exception_controller` configuration option, set it to `null` instead
+ * Deprecated the `ExceptionController` and all built-in error templates, use the error renderer mechanism of the `ErrorRenderer` component
+ * Deprecated loading custom error templates in non-html formats: 
```

Comment:
please mention explicitly that using custom error pages in __html__ using twig keeps working as before.